### PR TITLE
Exercise 8

### PR DESCRIPTION
**Pull request by kogoe1**

### CODE REVIEW

File: docker_push [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#Push built docker image to dockerhub
+echo "$DOCKER_PASSWORD" | docker login -u kogoe1 --password-stdin
+docker push kogoe1/todo_app:latest
+
+# Get the latest image from Docker Hub
+docker pull kogoe1/todo_app:latest
+docker login --username=fiifiogoe@yahoo.com --password=$HEROKU_API_KEY registry.heroku.com
```
Comment (BenRamchandani) [inline docker_push:8]: You don't need to put a username in here, `_` will do and Heroku will work it out from the API key

File: docker_push [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#Push built docker image to dockerhub
```
Comment (BenRamchandani) [inline docker_push:2]: It's worth having a `set -e` here so that the script exits if one of the lines fails.

**Review by BenRamchandani — COMMENTED** [review]
Looks good. I've left a couple of optional comments you can pick up on for the next module if you'd like.

File: docker_push (hunk 1 above)
Comment (kogoe1) [reply docker_push:8]: Thanks for the comments Ben. Your suggestion has been applied

File: docker_push (hunk 2 above)
Comment (kogoe1) [reply docker_push:2]: Good suggestion - thanks. Have added it

**Pull request merged**